webpack.config.js

```diff
@@ -149,7 +149,7 @@ module.exports = async function(_, env) {
         {
           // Emscripten modules don't work with Webpack's Wasm loader.
           test: /\.wasm$/,
-          exclude: /_bg\.wasm$/,
+          exclude: /(?<!\bpkg-parallel[\\/]squoosh_oxipng)_bg\.wasm$/,
```

Comment:
I find this bit confusing. Are we re-enabling Webpack’s Wasm loader for those two modules? If so, why?

Reply:
I agree it is confusing. I could clean it up if you want, but then you said we're migrating from Webpack anyway and there's no point in investing too much time to do it in a clean way?

Reply:
Fair.